### PR TITLE
A11y: use visually hidden styles rather than visibility, make visible on :focus as well

### PR DESCRIPTION
**Pull request by patrickhlauke**
- `visibility:hidden` also hides text from assistive technologies, leading to the settings link lacking an accessible name when not hovered (focused with keyboard only, for instance)
- turn the styles around and hide the non-hovered link's span with "visually hidden" styles https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/
- also include `:focus` to make the text visible when hovered (for sighted keyboard users)

Closes https://github.com/bitwarden/browser/issues/1989

### CODE REVIEW

File: src/popup/scss/pages.scss [hunk 1]
```diff
@@ -1,91 +1,96 @@
...
+            }
+        }
+
+        &:hover, &:focus {
+            text-decoration: none;
+
+            @include themify($themes) {
+                color: themed('primaryColor');
+            }
+
```
Comment (addisonbeck) [inline src/popup/scss/pages.scss:71]:
```suggestion
```

**Review by addisonbeck — COMMENTED** [review]
Thanks for the contribution! This looks verbatim the way ally suggests, so no issues with the content. I noticed an extra line break, so lets remove that. Can you attach a screenshot or short screen recording showing how this looks with your changes? After that, I think we can get this merged 🙂

**Comment by patrickhlauke** [comment]
quick video showing NVDA screenreader announcing the "Settings" link when only the cog is showing (see https://github.com/bitwarden/browser/issues/1989).

current (just navigating using `Tab`)

https://user-images.githubusercontent.com/895831/133849765-3f2671d4-e3c1-4b0b-bb6c-7eee3a495a90.mp4

after the fix (navigating using cursor/reading keys, then using `Tab` - note how "Settings" is now announced even when text is not visible, and how using keyboard focus now actually makes "Settings" text visible)

https://user-images.githubusercontent.com/895831/133851384-8ec42d77-bda7-498e-9a82-5ae4365b4772.mp4

**Comment by Hinton** [comment]
It seems `pages.scss` changed from using `LF` to `CRLF` for line-breaks.

**Comment by patrickhlauke** [comment]
@Hinton ah, apologies ... my fault for doing this PR while on vacation on a different machine that seems not to have been set up the way I normally like it. let me see if i can easily fix that and push a new commit

**Comment by patrickhlauke** [comment]
There. Thanks for the heads-up @Hinton

**Review by addisonbeck — APPROVED** [review]

**Comment by addisonbeck** [comment]
Thanks again @patrickhlauke 🎉

**Pull request merged**